ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestsecurity.py

```diff
@@ -36,3 +36,45 @@ def test_definition(self) -> dict | None:
         anta_tests.append({"VerifyAPIHttpsSSL": {"profile": profile, "result_overwrite": {"custom_field": f"eAPI HTTPS SSL Profile: {profile}"}}})
 
         return {self.anta_module: anta_tests}
+
+
+class AvdTestIPSecurity(AvdTestBase):
+    """
+    AvdTestIPSecurity class for IP security connection tests.
+    """
+
+    anta_module = "anta.tests.security"
+
+    @cached_property
+    def test_definition(self) -> dict | None:
+        """
+        Generates the proper ANTA test definition for all IP security connection tests.
+
+        Returns:
+            test_definition (dict): ANTA test definition.
+        """
+        anta_tests = []
+
+        # Check if there are any path groups with static peers
+        if (path_groups := get(self.structured_config, "router_path_selection.path_groups")) is None:
+            LOGGER.info("No router path-group configured to collect the static peer. %s is skipped.", self.__class__.__name__)
```

Comment:
Do you think this log message could cause confusion for users running validate state in a fabric without any WAN devices? They might think something is missing when it's not.

Reply:
The router path configuration is supported on fabric devices also so if there is no router path-group configured we are mentioning in log as to collect the static peer we need router path-group to be configured  
```
leaf1-dc1(config-peer-router-ip-10.1.1.1-Mahesh)#show active 
router path-selection
   path-group Mahesh
      peer static router-ip 10.1.1.1
```